plugins/ebitdo/ebitdo.quirk

```diff
@@ -31,3 +31,15 @@ USB\VID_2DC8&PID_9001=none
 # FC30_ARCADE
 USB\VID_8000&PID_1002=none
 USB\VID_2DC8&PID_1002=none
+
+#SF30 pro/sn30 pro
+##Nintendo Switch mode
+USB\VID_057E&PID_2009=none
+##Dinput mode
```

Comment:
Surely the only thing that should match is the Dinput mode? The other modes don't seem to implement the 8bitdo-firmware protocol.

Reply:
Well so actually `FU_EBITDO_USB_RUNTIME_EP_OUT` is in the descriptors for 3 of the 4 modes.  It's just missing for the macOS mode.  So that was making me think all the modes should still be matched.